POCServer/src/java/DAL/POCContext.java

```diff
@@ -0,0 +1,80 @@
+package DAL;
+
+import DAL.Public.Context;
+import Model.Roles;
+import Model.User;
+import Model.UserRoles;
+import java.util.List;
+
+public class POCContext implements Context {    
+    
+    //user
+    @Override
+    public List<User> getUsers() {
+        UserHelper userHelper = new UserHelper();
```

Comment:
I saw that you use the UserHelper in several places, please could you consider change the code to use a Singleton pattern to avoid create a new instance of UserHelper every time that a method is used.